extensions/extensions-helm/aissemble-mlflow-chart/README.md

```diff
@@ -13,8 +13,8 @@ The following properties are inherited from the base [MLflow chart](https://gith
 
 | Property | Description | Default |
 |----------|-------------|---------|
-| image.registry | MLflow image registry | NB: OSS: update with aissemble docker repository |
-| image.repository | MLflow image repository | boozallen/aissemble-mlflow |
+| image.registry | MLflow image registry | docker.io |
+| image.repository | MLflow image repository | bitnami/mlflow |

```

Comment:
A: Because this table is only overridden defaults, these rows should be removed entirely.

Reply:
Removed this from the table.